ModManager_Classes/Utils/ModinfoCreationManager.cs

```diff
@@ -43,41 +46,76 @@ public String ModDepsJoined
         }
         private String _mod_deps_joined;
 
-        public static ModinfoCreationManager Instance { get; } = new ModinfoCreationManager();
+        public ObservableCollection<DlcId> DLCs { get; set; } = new();
 
-        public ModinfoCreationManager()
+        public ModinfoFactory()
```

Comment:
I'd handle all GitHub actions warnings. VS should also show them all the time.

They only distract from important warnings.